### PR TITLE
New package: Samsung.SoftwareLauncher version 1.1.1.4

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

### CODE REVIEW

**Review by Trenly — CHANGES_REQUESTED** [review]
image

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

File: manifests/s/Samsung/SoftwareLauncher/1.1.1.4/Samsung.SoftwareLauncher.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Samsung.SoftwareLauncher
+PackageVersion: 1.1.1.4
+Installers:
```
Comment (stephengillie) [inline manifests/s/Samsung/SoftwareLauncher/1.1.1.4/Samsung.SoftwareLauncher.installer.yaml:6]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.DotNet.Framework.DeveloperPack_4
Installers:
```

**Comment by stephengillie** [comment]
Manual Validation failed with:
> 2023-07-21 15:20:59.127 [CLI ] Installer [X64,exe,Unknown,en-US] not applicable: Architecture was excluded by caller : X64  
> 2023-07-21 15:20:59.128 [CLI ] No suitable installer found for manifest Microsoft.DotNet.Framework.DeveloperPack_4 with version 4.8.1  
> 2023-07-21 15:20:59.128 [CLI ] Terminating context: 0x8a150104 at D:\a\_work\1\s\external\pkg\src\AppInstallerCLICore\Workflows\DependenciesFlow.cpp:126  

It appears that the dependency doesn't have a valid matching installer.

**Comment by upintheairsheep** [comment]
> Automatic Validation ended with:
> image

Dependencies required

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 2024-01-08 13:22:21.864 [CLI ] Starting: 'C:\Users\User\AppData\Local\Temp\WinGet\Microsoft.DotNet.Framework.DeveloperPack_4.4.8.1\ndp481-devpack-enu.exe' with arguments '/install /passive /norestart' 2024-01-08 13:27:47.651 [CLI ] ShellExecute installer failed: 3010

(Automated response - build 530.)

**Comment by stephengillie** [comment]
There seems to be an issue with the dependency `Microsoft.DotNet.Framework.DeveloperPack_4`.

**Pull request closed without merging**